vendor/templates/vendor/manage/config_list.html

```diff
@@ -0,0 +1,35 @@
+{% extends "vendor/manage/base.html" %}
+{% load i18n %}
+{% load admin_tags %}
+
+{% block title %}{{ title|default:"Config List" }}{% endblock %}
+
+{% block content %}
+<div class="card">
+    <div class="card-header">
+        {{ config_key }}
+    </div>
+    <div class="card-body">
+        <table class="table table-striped table-responsive-sm w-100">
+            <thead>
+                <tr>
+                    <th>Site</th>
```

Comment:
need trans tags for headers (site, value, new)

Reply:
trans tag added